bpf/bpf_overlay.c

```diff
@@ -687,7 +687,7 @@ int cil_from_overlay(struct __ctx_buff *ctx)
 		 * re-inserted from the stack.
 		 */
 		if (decrypted)
-			obs_point = TRACE_FROM_STACK;
+			obs_point = TRACE_FROM_CRYPTO;
```

Comment:
Just a note for 1.18: decrypted packets (previous ESP) will no longer get to from_overlay because of VinE.

Reply:
Yep, thank you very much for the note. Dropping the IPSec-related part from this PR and moving to a future one.